generated/thrift/rpc.thrift

```diff
@@ -92,6 +93,22 @@ struct WriteRequest {
 	3: required Datapoint datapoint
 }
 
+struct FetchMetadataBatchRawRequest {
+	1: required binary nameSpace
```

Comment:
nit: rename to `namespace`

Reply:
So the problem is that thrift does not allow the keyword `namespace` as its reserved... another reason to move to GRPC sometime in the future.. or rename namespace to be table for a shorter term fix, ha.

Reply:
haha damn, that's terrible